### ISSUE TITLE
Bestial Mutagen Claw Strike damage dice overriden by Clawed Catfolk Claw Strike

### ISSUE THREAD

**Issue by KonradMcClure** [foundryvtt/pf2e#15949]
Both Bestial Mutagen and the Clawed Catfolk Heritage provide a Claw attack. When under the effects of the Bestial Mutagen correctly gains the Jaws attack and +2 to attack rolls and athletics. However the heritage's Claw Strike appears to override the Mutagen's Claw Strike, as the image (should be bear claw), traits, and damage dice only show the heritage's stats.

However, I'm not sure if there are clear rules out there for how these two should interact. I'd rule the mutagen would override the heritage, but I'd need to do some rule searching to back that up. Alternatively it could just add the claw and jaws with a different label ("Claws (Bestial Mutagen)" for instance) and let the players and GM figure out how to rule from there.

**Comment by simonward** [comment]
Duplicate of https://github.com/foundryvtt/pf2e/issues/15421

**Comment by ammalagonc** [comment]
Closing as we will track this issue via #15421
